gui/system/migrations/0039_crash_reporting.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2019-03-25 03:24
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import freenasUI.freeadmin.models.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('system', '0038_merge_20190226_1008'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='settings',
+            name='stg_crash_reporting',
+            field=models.NullBooleanField(help_text='Enable sending anonymous crash reports to iXsystems', verbose_name='Crash reporting'),
```

Comment:
It should default to True for freenas, False for truenas.

Reply:
Fixed